scripts/install-chef-suse.sh

```diff
@@ -418,12 +418,14 @@ check_or_create_ptf_repository () {
         if ! [ -e "/srv/tftpboot/suse-$version/$arch/repos/$repo/repodata/repomd.xml" ]; then
             # Only do this for CROWBAR_FROM_GIT, as usually the crowbar package
             # creates the repo metadata for PTF
-            if [ -n $CROWBAR_FROM_GIT ]; then
+            if [ -n "$CROWBAR_FROM_GIT" ]; then
                 echo "Creating repo skeleton to make AutoYaST happy."
                 if ! [ -d /srv/tftpboot/suse-$version/$arch/repos/$repo ]; then
                     mkdir /srv/tftpboot/suse-$version/$arch/repos/$repo
                 fi
-                /usr/bin/createrepo /srv/tftpboot/suse-$version/$arch/repos/$repo
+                if ! /usr/bin/createrepo /srv/tftpboot/suse-$version/$arch/repos/$repo ; then
+                    die "Createrepo failed to create a repository in /srv/tftpboot/suse-$version/$arch/repos/$repo"
+                fi
```

Comment:
> TBH, I'm not sure we really need the `if [ -n "$CROWBAR_FROM_GIT" ]; then`

That came to my mind as well, because it wasn't an issue until now.
I just found these bugs while debugging something else.